src/plugins/dashboard/public/dashboard_app/top_nav/controls_toolbar_button/add_data_control_button.tsx

```diff
@@ -16,10 +16,10 @@ interface Props {
   controlGroup: ControlGroupContainer;
 }
 
-export const AddDataControlButton = ({ closePopover, controlGroup }: Props) => {
+export const AddDataControlButton = ({ closePopover, controlGroup, ...rest }: Props) => {
   return (
     <EuiContextMenuItem
-      key="addControl"
+      {...rest}
```

Comment:
This is following the [EUI guidelines for pass-through props](https://github.com/elastic/eui/blob/main/wiki/component-design.md#pass-through-props), which fixes the `EuiContextMenuPanel` from throwing an error where it **thinks** the items didn't have unique `key` properties because they were one layer too deep in the HTML.